README.md

```diff
@@ -28,16 +28,16 @@ Microsoft PowerToys is a set of utilities for power users to tune and streamline
 
 - Windows 11 or Windows 10 v2004 (19041) or newer.
 - Our installer will install the following items:
-   - [.NET 6.0.6 Desktop Runtime](https://dotnet.microsoft.com/download/dotnet/6.0#runtime-desktop-6.0.6) or a newer 6.0.x runtime.
+   - [.NET 6.0.7 Desktop Runtime](https://dotnet.microsoft.com/download/dotnet/6.0#runtime-desktop-6.0.7) or a newer 6.0.x runtime.
    - [Microsoft Edge WebView2 Runtime](https://go.microsoft.com/fwlink/p/?LinkId=2124703) bootstrapper. This will install the latest version.
    - [Microsoft Visual C++ Redistributable](https://docs.microsoft.com/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022) installer. This will install one of the latest versions available.
 
 ### Via GitHub with EXE [Recommended]
 
  [Microsoft PowerToys GitHub releases page][github-release-link], click on `Assets` at the bottom to show the files available in the release. Please use the appropriate the PowerToys installer that matches your machine's architecture.  For most people, it is `x64`.
  
- - **For x64 processors (most common):** click on `PowerToysSetup-0.60.1-x64.exe`
- - **For ARM64 processors:** `PowerToysSetup-0.60.1-arm64.exe`
+ - **For x64 processors (most common):** click on `PowerToysSetup-0.61.0-x64.exe`
+ - **For ARM64 processors:** `PowerToysSetup-0.61.0-arm64.exe`
```

Comment:
Maybe we could link the binaries directly here

Reply:
Sounds like a nice addition.